src/clj/test/cards/hardware.clj

```diff
@@ -680,6 +680,18 @@
       (prompt-choice :runner "Yes")  ; 6 installed
       (is (count-spy 6) "6 Spy Cameras installed"))))
 
+(deftest respirocytes-multiple
+  ;; Should draw multiple cards when multiple respirocytes are in play
+  (do-game
+   (new-game (default-corp)
+             (default-runner [(qty "Respirocytes" 9)]))
```

Comment:
I'd prefer for the test decks to follow deckbuilding rules as much as possible, particularly regarding max copies of cards. You can use the `(starting-hand)` utility to ensure there are 3 copies of Respirocytes in the opening hand.

Reply:
I think the problem is if the first or second damage hits the third respirocyte. Solution would be to just test for installing 2 I think (since then it's only possible you lose one.

Reply:
Makes sense.  I've switch it to 6 cards and doing 2 Respirocytes, since it's effectively testing the same thing.